source/cpu/cpuid_linux_s390x.go

```diff
@@ -96,4 +102,29 @@ func getCpuidFlags() []string {
 	return r
 }
 
-func getCpuidAttributes() map[string]string { return nil }
+func getCpuidAttributes() map[string]string {
+	ret := make(map[string]string)
+	ret["hypervisor"] = getHypervisor()
+
+	return ret
+}
+
+func getHypervisor() string {
+	data, err := os.ReadFile("/proc/sysinfo")
+	if err != nil {
+		klog.ErrorS(err, "failed to read /proc/sysinfo")
+		return ""
+	}
+
+	for _, line := range strings.Split(string(data), "\n") {
+		if strings.Contains(line, "Control Program:") {
+			parts := strings.SplitN(line, ":", 2)
+			if len(parts) == 2 {
```

Comment:
If this condition fails, should not you break the loop? I suppose there is only one `Control Program` line in `/proc/sysinfo`?

Reply:
I rework on it. Hope it make more sense. 
Thank you for reviewing.